packages/portal-api/server.js

```diff
@@ -9,6 +9,8 @@ const Toppsy = require('toppsy');
 const Vision = require('vision');
 const Pack = require('./package');
 const Portal = require('./lib');
+const path = require('path');
```

Comment:
please use uppercase external modules, makes it clear which variables are local and which are from external sources